README.md

```diff
@@ -3089,19 +3089,22 @@ class ApiLogger < Grape::Middleware::Base
 end
 ```
 
-## Before and After
+## Before, After and Finally
 
 Blocks can be executed before or after every API call, using `before`, `after`,
 `before_validation` and `after_validation`.
+If the API fails the `after` call will not be trigered, if you need code to execute for sure
+use the `finally`
 
 Before and after callbacks execute in the following order:
 
 1. `before`
 2. `before_validation`
 3. _validations_
-4. `after_validation`
-5. _the API call_
-6. `after`
+4. `after_validation` (if Validation Successful)
+5. _the API call_ (if Validation Successful)
+6. `after` (if Validation & API Successful)
+7. `finally` (ALWAYS)
```

Comment:
Nitpicking, let's not SHOUT :) I would make all of the comments lowercase (`upon successful validation`, `always`, etc.)